GoogleForCreators/web-stories-wp
# Use shortcode when classic editor plugin is active

## Summary

If a user has the classic-editor plugin active, the embed story block does not preview and seems broken. Instead of using the embed when classic-editor / TinyMCE is active, using the shortcode. See #3135

## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions
1. Activate classic editor plugin. 
2. Create a story.
3. Publish story. 
4. Click add post. 
5. Go to post edit screen, see shortcode.



---



Fixes #4833
Blocked #4780
